core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -81,6 +86,34 @@ def parse_row(row):
     return [lsn, backup_label, tablespace]
 
 
+def convert_to_ns(data):
+    """
+    This function is used as a replacement when system doesn't have ns native support
+    in time and os function.
+    """
+    return int(data * NANOSECONDS_PER_SECOND)
+
+
+def current_time_ns():
+    """
+    unified function to return current time in ns
+    """
+    try:
+        return time.time_ns()
+    except AttributeError:
+        return convert_to_ns(time.time())
+
+
+def mtime_ns(stat_obj):
+    """
+    unified function to return mtime in ns
+    """
+    try:
+        return stat_obj.st_mtime_ns
+    except AttributeError:
+        return convert_to_ns(stat_obj.st_mtime)
```

Comment:
`st_mtime_ns` exists since python 3.3.  Do we even support versions < 3.3 ? If we do, then add a comment so we know when we can remove it, otherwise we can probably just remove this check.

See: https://docs.python.org/3/library/os.html#os.stat_result.st_mtime_ns

Reply:
Good Catch.
I've simple removed the function, and adapted the 2 lines using it.